test cases/nasm/2 asm language/meson.build

```diff
@@ -1,4 +1,4 @@
-project('test', 'c')
+project('test', 'c', 'nasm')
```

Comment:
This test already do add_language() below, that's needed to skip the test on CI that does not have nasm installed.

Reply:
Fixed